opus/application/apps/help/templates/help/about.html

```diff
@@ -176,7 +176,7 @@
     We would also appreciate if you would drop us a note to let us know about it.
 </p>
 
-<p>Please note that OPUS works best with a browser size of at least 1280 by 720. We
+<p>Please note that OPUS works best with a browser size of at least 1280 by 275. We
```

Comment:
275 is not working the way I expected. Perhaps I screwed up using the inspector tool. The idea is that in the gallery view with the metadata detail modal popped up, it should fit completely on the screen. Otherwise, if the browser is too short, a bogus scrollbar shows up. So figure out how tall that is, round it to something pretty, and use that value.